doc/howtos/pull-request-guidelines.rst

```diff
@@ -168,3 +168,7 @@ Dos and Don'ts
   happens only for the full set of commits that are getting merged, so
   when bisecting history, the merge commits are good candidates for
   testing because they are more likely to build correctly.
+
+* Never add backdoors (= unexpected access to the system), debug features,
```

Comment:
Never add backdoors or debug features for unexpected access to the system to the default configuration.  Such capabilities may be enabled only in explicit debug configurations of the OS images.